controllers/control_plane_init_locker.go

```diff
@@ -77,55 +90,55 @@ func (l *controlPlaneInitLocker) Acquire(cluster *clusterv2.Cluster) bool {
 				},
 			},
 		},
+		Data: map[string]string{
+			lockInformationKey: string(b),
```

Comment:
Would it be worth it to have a "setter"  method similar to readLockInformation here?

Reply:
i could do writeLockInformation to wrap the json stuff in one place